website/content/en/preview/tasks/scheduling.md

```diff
@@ -75,6 +75,25 @@ While `nodeSelector` is like node affinity, it doesn't have the same "and/or" ma
 So all key-value pairs must match if you use `nodeSelector`.
 Also, `nodeSelector` can do only do inclusions, while `affinity` can do inclusions and exclusions (`In` and `NotIn`).
 
+### Supported Labels
+The following labels are supported by Karpenter. They may be specified as provisioner requirements or pod scheduling constraints.
+
+| Label                                             | Example    | Description                                                                                                                                 |
+| ------------------------------------------------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| kubernetes.io/os                                  | linux      | Operating systems include `linux`, `windows`                                                                                                |
+| kubernetes.io/arch                                | amd64      | Architectures include `amd64`, `arm64`                                                                                                      |
+| node.kubernetes.io/instance-type                  | p3.8xlarge | Instance types are defined by your cloud provider ([aws](https://aws.amazon.com/ec2/instance-types/))                                       |
+| topology.kubernetes.io/zone                       | us-west-2a | Zones are defined by your cloud provider ([aws](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html)) |
+| karpenter.sh/capacity-type                        | spot       | Capacity types include `spot`, `on-demand`                                                                                                  |
+| karpenter.amazonaws.com/instance.family           | p3         | [AWS Specific] Instance types of similar properties but different resource quantities                                                       |
```

Comment:
https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-metadata-name
https://github.com/NVIDIA/gpu-feature-discovery#verifying-everything-works

It's typically used to represent properties of objects. In some sense, it could map to a JQ query :P

Reply:
fair enough :)